.semgrep/custom-rules.yml

```diff
@@ -162,7 +162,7 @@ rules:
   message: String formatted before gettext
   patterns:
     - pattern-either:
-        - pattern-regex: |
-            _\(\s*"(.*)".format\(
-        - pattern-regex: |
+        - pattern-regex: |-
```

Comment:
This is known as a block chomping indicator; see https://yaml-multiline.info/. Without it, a newline may be added to the end of the string, which causes the regular expression to not behave as expected. In my testing, this particular rule only fired if I removed the subsequent one and ensured no newline was at the end of the file.